src/util/LocalStorage.js

```diff
@@ -0,0 +1,10 @@
+const loadFromLocalStorage = key => {
+  const value = localStorage.getItem(key);
+  return JSON.parse(value);
```

Comment:
`JSON.parse()` can fail and throw an error when called with invalid JSON data. You need to handle this situation with a `try/catch` block.